### ISSUE TITLE
nitpick: wrong delay?

### ISSUE THREAD

**Issue by deshipu** [adafruit/Adafruit_CircuitPython_ILI9341#3]
Sorry, I just looked at the code and saw this:
```
b"\x01\x80\x80"# Software reset then delay 0x78 (120ms)
```

I assume that ``\x80`` is the command to do a delay, because later on there are lines that say:

```
b"\x11\x80\x78"# Exit Sleep then delay 0x78 (120ms)
```

But if that's true, then it seems that command delays for 128ms, not 120ms as the comment says.

**Comment by makermelissa** [comment]
We'd have to look at the data sheet and test it out to be sure. I do know that it works perfectly the way it currently is. It's possible somebody changed the value without changing the comment.

**Comment by deshipu** [comment]
If it works, then we could simply change the comment to reflect the change, though I'm not sure 8ms would make a huge difference — I suspect a typo.

**Comment by makermelissa** [comment]
After looking at the data sheet, I suspect this is merely a copy/paste error and fixing the comment would work fine.

**Comment by dhalbert** [comment]
The original code was written in a hurry and then not used at all for a while, so it's not surprising there are minor bugs or inconsistencies. A similar init sequence is here, for the PyPortal:
https://github.com/adafruit/circuitpython/blob/master/ports/atmel-samd/boards/pyportal/board.c

Interestingly, the PyPortal version doesn't have software reset line at the top that this one does. @tannewt may have a comment on the differences.

**Comment by makermelissa** [comment]
> Interestingly, the PyPortal version doesn't have software reset line at the top that this one does. @tannewt may have a comment on the differences.

Yeah, neither does the Arduino Init code. However, it may have something to do with soft-resetting the MicroController still allowing displayio to show the error on screen even before it's reinitialized. We can try it without and see if it works just fine.

**Comment by makermelissa** [comment]
I just tested and it works fine both with or without the reset and delay.

**Comment by makermelissa** [comment]
I would like to see what @tannewt says about this init code before making any changes one way or the other.

**Comment by tannewt** [comment]
+1 to changing the comment. I don't think the actual delay amount matters as long as it's long enough.

The added software reset is meant to be tidier with display state by always starting from the same state. I think it's good to have but can't think of a specific bug it fixed.

**Comment by makermelissa** [comment]
Fixed via #4.
